test-butler-app/src/main/java/com/linkedin/android/testbutler/ButlerService.java

```diff
@@ -53,6 +59,56 @@ public boolean setWifiState(boolean enabled) throws RemoteException {
         }
 
         @Override
+        public boolean setGsmState(boolean enabled) throws RemoteException {
```

Comment:
Please move the logic for this into it's own class/file, similar to the other classes used here.